website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -972,10 +972,12 @@ A `upgrade_settings` block supports the following:
 
 * `node_soak_duration_in_minutes` - (Optional) The amount of time in minutes to wait after draining a node and before reimaging and moving on to next node. Defaults to `0`. <!-- The 0 default happens in code, not in Schema -->
 
-* `max_surge` - (Required) The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade.
+* `max_surge` - (Optional) The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade.
```

Comment:
Is this still Required?

Reply:
You're right, it's Optional, I'll fix it.